src/services/notifications/index.ts

```diff
@@ -520,3 +541,68 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }),
   }
 }
+
+export const getPushNotificationContent = ({
+  type,
+  amountBaseCurrency,
+  baseCurrency,
+  amountDisplayCurrency,
+  displayCurrency,
+  userLanguage,
+}: {
+  type: NotificationType | "balance"
+  amountBaseCurrency: Satoshis | UsdCents
+  baseCurrency: WalletCurrency
+  amountDisplayCurrency?: DisplayCurrencyBaseAmount
+  displayCurrency?: DisplayCurrency
+  userLanguage?: UserLanguage
+}): {
+  title: string
+  body: string
+} => {
+  const locale = userLanguage || defaultLocale
+  const notificationType = type === "balance" ? type : `transaction.${type}`
+  const title = i18n.__(
+    { phrase: `notification.${notificationType}.title`, locale },
+    { walletCurrency: baseCurrency },
+  )
+  const baseCurrencyName = baseCurrency === WalletCurrency.Btc ? "sats" : ""
+  const baseCurrencySymbol = baseCurrency === WalletCurrency.Usd ? "$" : ""
+  const amount =
```

Comment:
Might consider renaming this to something like `displayedBaseAmount` to clarify that we're explicitly converting from the usual base amount unit (cents) to a unit for display purposes (dollars)

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/1338/commits/4e304d4a71fd4eb5d775bf0de2d020d0efa013df#diff-a6381842e83d5711a96e584a1b5651ebbf4a3cf46e7e66a85d1df9bac42b2a0fR569